### ISSUE TITLE
Logging changes (fine tuning)

### ISSUE THREAD

**Issue by 3eka** [030/n3dr#270]
**What would you like to be added**:
I would like to have an option to fine tune logging (like: INFO, WARN, ERROR, etc -- with NONE option too).
Or, some errors I want to see on the screen (like ERROR), other can go into file (like INFO ones)

**Why is this needed**:
Because, it is really annoying. Few examples:

- This is output (on the screen) when `--basePathPrefix` switch is used

```
++++++++++++++++++INFO[0240]/n3dr/internal/pkg/connection/connection.go:23 github.com/030/n3dr/internal/pkg/connection.(*Nexus3).Client() adding 'nexus' as a prefix to the basePath
*********++++++++++INFO[0240]/n3dr/internal/pkg/connection/connection.go:23 github.com/030/n3dr/internal/pkg/connection.(*Nexus3).Client() adding 'nexus' as a prefix to the basePath
*INFO[0240]/n3dr/internal/pkg/connection/connection.go:23 github.com/030/n3dr/internal/pkg/connection.(*Nexus3).Client() adding 'nexus' as a prefix to the basePath
*******+++++++++++++*++**INFO[0241]/n3dr/internal/pkg/connection/connection.go:23 github.com/030/n3dr/internal/pkg/connection.(*Nexus3).Client() adding 'nexus' as a prefix to the basePath
++++++INFO[0241]/n3dr/internal/pkg/connection/connection.go:23 github.com/030/n3dr/internal/pkg/connection.(*Nexus3).Client() adding 'nexus' as a prefix to the basePath
*********++++++++++INFO[0241]/n3dr/internal/pkg/connection/connection.go:23 github.com/030/n3dr/internal/pkg/connection.(*Nexus3).Client() adding 'nexus' as a prefix to the basePath
*INFO[0241]/n3dr/internal/pkg/connection/connection.go:23 github.com/030/n3dr/internal/pkg/connection.(*Nexus3).Client() adding 'nexus' as a prefix to the basePath
...
```
As you might see _n3dr_ keeps repeating following, every time when it advances to the next item, which make it hard to follow:
* INFO[xxxx]/n3dr/internal/pkg/connection/connection.go:23 github.com/030/n3dr/internal/pkg/connection.(*Nexus3).Client() adding 'nexus' as a prefix to the basePath

```
:; n3dr --version
n3dr version 6.7.3
:; go version
go version go1.19.1 illumos/amd64
```

**Steps**:

- [ ] Write to a log file, e.g.: `/var/log/n3dr/<date-time>.log`
- [ ] Revise current log levels (trace, debug, info, warn, error, fatal, panic) / Will log anything that is info or above (warn, error, fatal, panic). Default. log.SetLevel(log.InfoLevel)
- [ ] log.SetFormatter(&log.JSONFormatter{}) in production
- [x] Set multiple log levels: log.Trace("Something very low level.")
log.Debug("Useful debugging information.")
log.Info("Something noteworthy happened!")
log.Warn("You should probably take a look at this.")
log.Error("Something failed but I'm not quitting.")
None
- [ ] decide what goes into stdout and stderr
- [ ] only show errors in console and other logs go into file

**Comment by 030** [comment]
6.8.0

**Comment by 030** [comment]
@3eka Did you try the logging option?

**Comment by 030** [comment]
@3eka Could you create another issue to address "the raw repository and FATA[2022-11-04T16:30:10+01:00]/n3dr/cmd/n3dr/repositoriesV2.go:70 main.glob..func7() parse ":0/v2/_catalog": missing protocol scheme" issue?

**Comment by 030** [comment]
Example: `n3dr repositoriesV2 --backup -u some-user -p some-pass -n localhost:9000 --https=false --directory-prefix /tmp/some-dir --dockerPort 9001 --dockerHost http://localhost --logFile some-file.log --logLevel trace --syslog`

**Comment by 030** [comment]
7.0.0

**Comment by 3eka** [comment]
Sorry, I have missed your latest update completely.

Let us focus on logging:
* if I choose logFile and logLevel, I expected that everything is logged there
* in reality, tool still logs to stdout

Thanks and regards

**Comment by 3eka** [comment]
HI @030 ,

can you describe a bit logging logic?

I am expecting following:
* When `--logFile=/path/to/log/file` is specified, ALL output should end there. Right now, tool sends to stdout (even with `--logLevel=none` settings):
```
INFO[0000] n3drHomeDir: '/export/home/predrag_zecevic/.n3dr' 
INFO[0000] Using config file: '/export/home/predrag_zecevic/.n3dr/config.yml' 
```
Rest is sent to log file
* If I stop n3dr and start again, with same logging options (e.g. logging to the same file), new content is added to the existing file (I would expect overwrite -- or another switch which can be used for automating this, like `--logMethod={new,overwrite}`)

I still have to check all logging level options
Regards.
